cpp/src/plasma/protocol.h

```diff
@@ -28,6 +28,12 @@ namespace plasma {
 
 using arrow::Status;
 
+template <class T>
+bool verify_flatbuffer(T* object, uint8_t* data, size_t size) {
```

Comment:
If you receive `int64_t size` here, then you only have to static cast to `size_t` in one place when you create the verifier